GOVERNANCE.md

```diff
@@ -0,0 +1,152 @@
+# Prometheus Governance v1.0
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Prometheus community. Common terminology used in this governance document are listed below:
+
+* **Team members**: Any members of the private [prometheus-team][team] Google group.
+
+* **Maintainers**: Team members who are leads on an individual project ([MAINTAINERS.md][maintainers.md]).
```

Comment:
I assume the definition here has to change.

Reply:
What exactly are you referring to? Should we add "or parts thereof"? Incidentally, this definition here never included team membership.

Reply:
Sorry, I cannot read. :-(
Will fix.

Reply:
Thanks!